build.gradle

```diff
@@ -441,6 +450,14 @@ allprojects {
       }
     }
 
+    applyScalaConfig = { ->
+      spotless {
```

Comment:
Spotless is Gradle plugin which provides support for many language-specific formatting tools... like `scalafmt` for Scala (which is itself completely independent from Gradle)